src/screens/ENSAssignRecordsSheet.js

```diff
@@ -49,7 +54,10 @@ export const BottomActionHeight = ios ? 270 : 250;
 
 export default function ENSAssignRecordsSheet() {
   const { colors } = useTheme();
-  const { name } = useENSProfile();
+  const { name, mode } = useENSProfile({
+    setExistingRecordsWhenInEditMode: true,
+  });
+  useENSProfileForm({ createForm: true });
```

Comment:
it took me a minute to follow the path for this hook, this is not specific to this PR but feels like we can arrange them in a better way
for example here we're not getting anything from `useENSProfileForm` but we're calling `useENSProfile` from the `useENSProfileForm` hook and also from the component itself